metron-platform/metron-parsers/README.md

```diff
@@ -45,6 +45,33 @@ There are two general types types of parsers:
       * `ERROR` : Throw an error when a multidimensional map is encountered
     * `jsonpQuery` : A [JSON Path](#json_path) query string. If present, the result of the JSON Path query should be a list of messages. This is useful if you have a JSON document which contains a list or array of messages embedded in it, and you do not have another means of splitting the message.
     * A field called `timestamp` is expected to exist and, if it does not, then current time is inserted.  
+
+## Parser Error Routing
+
```

Comment:
It feels like it may be confusing.  The title is `Parser Error Routing`, but the description is `invalid`.  Then we talk about `invalid` again, but end with `error` messages.

We may need to be more clear about errors v. invalids, and how they both can be routed to the error queue.  Or not use the word invalid but instead us a more specific error, like saying there are parser errors and validation errors.